plans/tier1.fmf

```diff
@@ -185,3 +184,20 @@ discover+:
     - name: reboot machine
       how: ansible
       playbook: tests/ansible_collections/reboot.yml
+
+/convert_offline_systems:
+  discover+:
+    test: checks-after-conversion
+  prepare+:
+    - name: install subscription manager
+      how: ansible
+      playbook: tests/ansible_collections/roles/install-submgr/main.yml
+    - name: allow access to Satellite only
+      how: shell
+      script: pytest -svv tests/integration/tier1/convert-offline-systems/prepare_system.py
+    - name: run offline conversion
+      how: shell
+      script: pytest -svv tests/integration/tier1/convert-offline-systems/run_conversion.py
+    - name: reboot after conversion
+      how: ansible
+      playbook: tests/ansible_collections/reboot.yml
```

Comment:
Please make the indention same as previous sections

Reply:
Thanks, I fixed the indention also in previous section here: https://github.com/oamg/convert2rhel/pull/310/files#diff-effa0515e248da3fe738adfe9c21f2977701c9e1c8701bcc86c36acb32415a43R175